.gitignore

```diff
@@ -8,8 +8,9 @@
 logs
 
 # build artifacts
-sermant-agent-*
+sermant-agent*
```

Comment:
It cannot be changed to sermant-agent*. If it is changed to sermant-agent*, the change of sermant-agentcore  will be ignored.